.github/workflows/load-test-oha.yml

```diff
@@ -0,0 +1,83 @@
+name: Load Test with oha CLI
+
+on:
+  push:
+    branches:
+      - '*'
+
+jobs:
+  load-test:
+    runs-on: ubuntu-latest
+
+    # env:
+    #   PYTHONPATH: ${{ github.workspace }}
+
+    steps:
+      - name: Checkout code
```

Comment:
@YeagerAI-Bulat since we are doing these same steps on other workflows, it would be great to abstract them to an action or similar.

Reply:
I have abstracted load tests and made it as a reusable workflow now

Reply:
@YeagerAI-Bulat I think that Cristiam was referring to the `checkout + .env copy + sed + OpenAI key` steps